test/functional/FunctionalCodeAction.hs

```diff
@@ -421,6 +424,7 @@ typedHoleTests = testGroup "typed hole code actions" [
 
       , testCase "shows more suggestions" $
             runSession hlsCommand fullCaps "test/testdata" $ do
+                disableWingman
```

Comment:
Can we have a new test to check that Wingman really disables hole fits? 

Otherwise we will not know if we accidentally break the dynflags config.